sedai/README.md

```diff
@@ -0,0 +1,53 @@
+# Autonomous Cloud Application Management
+
+*Autonomous is your future; Production is the new playground*
+
+## **[Schedule a Demo][2]**
```

Comment:
```suggestion
```
Let's have this section in one place only

Reply:
done